installation/aws/VmClarity.cfn

```diff
@@ -477,28 +477,26 @@ Resources:
                     ##
                     ## Orchestrator configuration
                     ##
-                    # Host for the VMClarity backend server
-                    APISERVER_HOST=apiserver
-                    # Port number for the VMClarity backend server
-                    APISERVER_PORT=8888
+                    # VMClarity API server address
+                    VMCLARITY_ORCHESTRATOR_APISERVER_ADDRESS=http://__CONTROLPLANE_HOST__:8888
```

Comment:
Did you check that the orchestrator and the apiserver can communicate on the same host with the node public ip?

Reply:
Yup, I've tested it. However the `__CONTROLPLANE_HOST__` variable is [substituted to the private IPv4](https://github.com/openclarity/vmclarity/blob/refactor-orchestrator-config/installation/aws/VmClarity.cfn#L210-L211) address of the host and not public.

It might be better to revert this config to `apiserver:8888` as all the VMClarity services are deployed in the same Docker network. Just to avoid unnecessary NAT in API <-> Orchestrator network comm.

Long term I'd probably deploy these services separately as systemd units with host networking.

Reply:
I think long term we need to understand if the orchestrator and the apiserver are deployed on the same host, and giving it address accordingly.

Reply:
BTW, I see you left it with the "VMCLARITY_ORCHESTRATOR_APISERVER_ADDRESS=http://__CONTROLPLANE_HOST__:8888" 
Is that intentional, or you missed it?

Reply:
> BTW, I see you left it with the "VMCLARITY_ORCHESTRATOR_APISERVER_ADDRESS=http://**CONTROLPLANE_HOST**:8888" Is that intentional, or you missed it?

Yeah, I wanted to discuss the course we should take here before reverting this change.

Reply:
@fishkerez Updated the PR with reverting to using the `apiserver` as host in teh Orchestrator config.